docs/source/suite.rst

```diff
@@ -320,142 +399,150 @@ Objective:
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| 1000 Obj. Rotation     | ``myoHandReorientOOD-v0``        | Hard            |         ✓        |      ✓        |          ✓          |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Leg walk               | ``myoLegWalk-v0``                | Easy            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Leg walk Rough Ground  | ``myoLegRoughTerrainWalk-v0``    | Hard            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Leg walk Hilly Ground  | ``myoLegHillyTerrainWalk-v0``    | Hard            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Leg walk Stair Ground  | ``myoLegStairTerrainWalk-v0``    | Hard            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Grasping & Placing     | ``myoChallengeRelocateP1-v0``    | Easy            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-| Chase Tag              | ``myoChallengeChaseTagP1-v0``    | Easy            |         ✓        |      ✓        |                     |
-+------------------------+----------------------------------+-----------------+------------------+---------------+---------------------+
-
-
-Variations:
+.. Sarcopenia
+.. ==============
+
+.. Sarcopenia is a muscle disorder that occurs commonly in the elderly population (Cruz-Jentoft and Sayer (2019))
+.. and characterized by a reduction in muscle mass or volume.
+.. The peak in grip strength can be reduced up to 50% from age 20 to 40 (Dodds et al. (2016)).
+.. We modeled sarcopenia for each muscle as a reduction of 50% of its maximal isometric force.
+
+
+.. .. _fatigue:
+
+.. Fatigue
+.. ============================
+.. Muscle Fatigue is a short-term (second to minutes) effect that happens after sustained or repetitive voluntary movement
+.. and it has been linked to traumas e.g. cumulative trauma disorder (Chaffin et al. (2006)).
+.. A dynamic muscle fatigue model (Ma et al. (2009)) was integrated into the modeling framework.
```

Comment:
@Vittorio-Caggiano -- we recently updated our fatigue model. Tagging you to quickly confirm that this is updated.